src/icon-button.jsx

```diff
@@ -100,6 +100,11 @@ const IconButton = React.createClass({
     onMouseLeave: React.PropTypes.func,
 
     /**
+     * Callback function for when mouse goes out of element it works with disabled element
+     */
+    onMouseOut: React.PropTypes.func,
```

Comment:
I don't think we need to expose this. it's super implementation detail... -ish :sweat_smile: